versioned_docs/version-2.0.0/keploy-explained/debugger-guide.md

```diff
@@ -44,12 +44,16 @@ Navigate to `launch.json` to begin crafting JSON objects.
       "asRoot": true,
       "console": "integratedTerminal",
       "program": "main.go",
-      "args": ["test", "-c", "<path_to_executable>"]
+      "args": ["test", "-c", "<path_to_executable>"],
+      "env": {
+        "PATH": "<path_to_Go_binary>"
+      }
     }
```

Comment:
only for one? we need record and test both

Reply:
Done. Please tell if any changes are required.